front/public/locales/fr/operationalStudies/scenario.json

```diff
@@ -50,7 +50,12 @@
     "duplicate": "Dupliquer",
     "filterLabel": "Nom, étiquette",
     "importTrainSchedule": "Importer",
-    "invalidRollingStock": "Matériel invalide",
+    "invalid": {
+      "pathfinding_not_found": "Chemin introuvable",
+      "pathfinding_failed": "Calcul interrompu",
+      "simulation_failed": "Simulation impossible",
+      "rolling_stock_not_found": "MR non trouvé"
```

Comment:
Is "MR" explicit enough? Is it a commonly used acronym?

Reply:
MR has been validated by @thibautsailly